components/textbook-beta/RelatedMaterialSection.vue

```diff
@@ -0,0 +1,104 @@
+<template>
+  <section class="related-material-section">
+    <h2>
+      Related Materials
+    </h2>
+    <div class="related-material-section__cards">
+      <AppDescriptionCard
+        v-for="{ title, description, segment, url } in prerequisites"
+        :key="title"
+        :description="description"
+        class="related-material-section__card"
+        :cta="{ label: 'Go to page', segment, url}"
+        :title="title"
+      />
+    </div>
+  </section>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component } from 'vue-property-decorator'
+import { CtaClickedEventProp } from '~/constants/segment'
+
+interface Prerequisite {
+  title: string;
+  description: string;
+  segment: CtaClickedEventProp;
+  url: string;
+}
+
+@Component
+export default class RelatedMaterialSection extends Vue {
+  prerequisites: Prerequisite[] = [
+    {
+      title: 'Introduction to Quantum Computing and Quantum Hardware',
+      description:
+        'The Qiskit Global Summer School 2020 coursework, lab, and lecture materials are available online.',
```

Comment:
I agree @korgan00 - there's a lot of opportunity for refactors. I can create issues for each component that we should abstract/refactor.

But since this task was supposed to be a `size - small` — I'm leaving out a lot of the refactoring that I noticed, was needed.

Once all the reviews are done, I'll include the refactor issues here